src/robot-handler.cpp

```diff
@@ -119,7 +70,7 @@ namespace simple_mpc
     std::vector<std::string> joint_names;
```

Comment:
`GetControlledJointNames` can also be deleted since it now directly corresponds to `model_.names` (or getModel().names)